datalad/interface/clean.py

```diff
@@ -56,32 +57,44 @@ class Clean(Interface):
       $ datalad clean
     """
 
+    result_renderer = 'tailored'
+
     _params_ = dict(
         dataset=Parameter(
             args=("-d", "--dataset"),
             doc="""specify the dataset to perform the clean operation on.  If
                 no dataset is given, an attempt is made to identify the dataset
                 in current working directory""",
             constraints=EnsureDataset() | EnsureNone()),
-        # TODO:  --info  -- which performs dry run just summarizing what is to be cleaned up
+        info=Parameter(
+            args=("-i", "--info"),
+            doc="""Report on cleanable locations - not actually cleaning up
+            anything.""",
+            action='store_true'),
         # TODO: Python only???
         what=Parameter(
             args=("--what",),
             dest='what',
-            choices=('cached-archives', 'annex-tmp', 'annex-transfer', 'search-index'),
+            choices=('cached-archives', 'annex-tmp', 'annex-transfer',
+                     'search-index'),
             nargs="*",
-            doc="""What to clean.  If none specified -- all known targets are
-            cleaned"""),
+            doc="""What to clean or report on. If none specified -- all known
```

Comment:
If `--dry-run` is adopted, this first half of the doc change can be removed again, I think.

Reply:
Agree.

Reply:
Done